storages/backends/azure_storage.py

```diff
@@ -88,6 +88,7 @@ def _save(self, name, content):
         else:
             content_type = mimetypes.guess_type(name)[0]
 
+        content.seek(0)
         if hasattr(content, 'chunks'):
             content_data = b''.join(chunk for chunk in content.chunks())
```

Comment:
As far as I understand `content` is a file-like object, but I have no idea about this case. Is it something Azure-specific? Is it ok to call `seek` on `content ` before this?